src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -193,6 +201,11 @@ protected WebResourceResponse interceptRequest(String rawUrl) {
       if (path.equalsIgnoreCase("/webxdc.js")) {
         InputStream targetStream = getResources().openRawResource(R.raw.webxdc);
         return new WebResourceResponse("text/javascript", "UTF-8", targetStream);
+      } else if (path.equalsIgnoreCase("/sandboxed_iframe_rtcpeerconnection_check_5965668501706.html")) {
+        // It is important to NOT return a malicious file instead of this one.
+        // Consider `iframe.srcdoc` as an alternative.
```

Comment:
@WofWca What does this comment mean, I don't understand it.

Reply:
Sorry, I meant to tell to be careful about changing this and surrounding code. Maybe adding this to the comment could help.

Reply:
i removed the comment with the recent commit as it seems a bit confusing - and also to to not duplicate the comment as it at least partly true also for the  other files.
i also think, it is pretty clear that all that code is sensible :) - otherwise the code is mostly self-explaining.
please feel free to add an updated comment if you still feel it is needed.